app/version-check/server/functions/checkVersionUpdate.js

```diff
@@ -62,7 +62,7 @@ export default () => {
 					msg: `*${ TAPi18n.__('Rocket_Chat_Alert', adminUser.language) }:*\n\n*${ TAPi18n.__(alert.title, adminUser.language) }*\n${ TAPi18n.__(alert.text, ...alert.textArguments || [], adminUser.language) }\n${ alert.infoUrl }`,
 				})),
 			banners: alerts.map((alert) => ({
-				id: `alert-${ alert.id }`,
+				id: `alert-${ alert.id }`.replace(/\./g, '_'),
```

Comment:
Needed because otherwise with a dot its inserting and nesting.  So like in the version one it was nesting like:
`versionUpdate-3: {
 '3': {
    '0': banner
   }
}
```